### PR TITLE
Russian translation added

### PR DESCRIPTION
**Pull request by klimgeran**
🇷🇺 Russian

### CODE REVIEW

File: src/config/i18n/ru.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    "addressResolutionEntries": "Стоимость",
+    "unresolved": "Неразрешенный",
+    "roles": "Роли",
+    "network": "Сеть",
+    "host": "IP/ DNS",
+    "port": "Порт",
+    "friendlyName": "Дружественное имя",
+    "direction": "направление",
+    "approximateExpired": "Приблизительный Срок Окончания",
+    "expiredInBlock": "Истек Срока в Блоке",
```
Comment (OlegMakarenko) [inline src/config/i18n/ru.json:149]: "Истечение Срока в Блоке"

File: src/config/i18n/ru.json [hunk 2]
```diff
@@ -0,0 +1,153 @@
+{
+    "langName": "Russian",
+    "age": "Возраст",
+    "aliasAction": "Действие Псевдонима",
+    "linkAction": "Действие Связи",
+    "amount": "Сумма",
+    "relativeAmount": "Относительная Сумма",
+    "height": "Высота",
+    "fee": "Плата",
```
Comment (OlegMakarenko) [inline src/config/i18n/ru.json:9]: Maybe "Комиссия" fits better here?

File: src/config/i18n/ru.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    "roles": "Роли",
+    "network": "Сеть",
+    "host": "IP/ DNS",
+    "port": "Порт",
+    "friendlyName": "Дружественное имя",
+    "direction": "направление",
+    "approximateExpired": "Приблизительный Срок Окончания",
+    "expiredInBlock": "Истек Срока в Блоке",
+    "index": "No.",
+    "Network Fees": "Рекомендуемый Мультиплеер Комиссий",
```
Comment (OlegMakarenko) [inline src/config/i18n/ru.json:151]: "Множитель"?

**Comment by klimgeran** [comment]
Thanks Oleg for the amendment, I changed, now the text is better

File: src/config/i18n/ru.json (hunk 2 above)
Comment (klimgeran) [reply src/config/i18n/ru.json]: Corrected for "Комиссия"

File: src/config/i18n/ru.json (hunk 1 above)
Comment (klimgeran) [reply src/config/i18n/ru.json]: Fixed

File: src/config/i18n/ru.json (hunk 3 above)
Comment (klimgeran) [reply src/config/i18n/ru.json]: Fixed :" Множитель"

**Comment by klimgeran** [comment]
#update ru.json

**Comment by klimgeran** [comment]
Welcome, I was happy to help Oleg! I wish you a good mood! 👍🏻🙂

**Pull request merged**